pkgs/development/compilers/cudatoolkit/common.nix

```diff
@@ -208,6 +214,9 @@ stdenv.mkDerivation rec {
   # --force-rpath prevents changing RPATH (set above) to RUNPATH.
   postFixup = ''
     addOpenGLRunpath --force-rpath {$out,$lib}/lib/lib*.so
+    ${lib.optionalString (lib.versionAtLeast version "11") ''
+      addOpenGLRunpath $out/bin/compute-sanitizer
```

Comment:
Does nsys not need `addOpenGLRunpath` as well?

Btw, I wonder if `addOpenGLRunpath $out/bin/*` would be smart enough to do the right thing?